Patches/Colony Leadership/WarriorLeadership.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<Operation Class="PatchOperationFindMod">
+		<mods>
+		<li>Colony Leadership</li>
+		</mods>
+
+		<match Class="PatchOperationSequence">
+			<operations>
+
+				<li Class="PatchOperationReplace">
+					<xpath>/Defs/HediffDef[defName="leader2"]/stages/li[1]/statOffsets</xpath>

```

Comment:
Why not just make this an Add operation, instead of Replace?

Reply:
Because the aura of warrior adds this

		  <AccuracyTouch>+0.15</AccuracyTouch>
		  <AccuracyShort>+0.15</AccuracyShort>		  
		  <AccuracyMedium>+0.15</AccuracyMedium>

I would need several operations to remove these non-functional attributes, faster to just replace the entire thing.

Reply:
Ah, understandable.